exchangelib/fields.py

```diff
@@ -529,31 +531,44 @@ def to_xml(self, value, version):
         return set_xml_value(field_elem, base64.b64encode(value).decode('ascii'), version=version)
 
 
-class DateField(FieldURIField):
-    value_cls = EWSDate
-
+class GenericDateField(FieldURIField):
     def from_xml(self, elem, account):
         val = self._get_val_from_elem(elem)
+        is_all_day = elem.find('{http://schemas.microsoft.com/exchange/services/2006/types}IsAllDayEvent')
```

Comment:
interesting, is there a file that stores these urls or something? it feels a bit out of place but if not, then I'm cool with keeping it here!

Reply:
Oh ya this was a total hack. There is no file that has these in their entirety, instead I followed the logic for how the fields are fetched, which happens by calling the `response_tag` function on instance of the `FieldUriField` class and dynamically generating '{%s}%s' % (TNS, self.field_uri_postfix) where TNS is the http part and the field_uri_postfix is what comes after the `:` in the field_uri param to the field  `BooleanField('is_all_day', field_uri='calendar:IsAllDayEvent', is_required=True, default=False)` for example. 

Normally a field is creating its own response tag. In this code, however, I want the GenericDateField to create to find a different field (is_all_day) and thus it has to generate the response tag this way. 

I can clean this up by referencing TNS.